monitoring/example_monitoring.log

```diff
@@ -0,0 +1,22 @@
+--- General Information ---
+#CPU: 1
+Total Memory: 1700
+Total Disk space: 10190136
+
```

Comment:
I assume 1700Gb and 10190136kb disk?

Reply:
never mind, I saw another log file, 1700MB

Reply:
Good suggestion; I made this explicit.